opm/parser/eclipse/EclipseState/Runspec.hpp

```diff
@@ -44,12 +45,13 @@ std::ostream& operator<<( std::ostream&, const Phase& );
 class Phases {
     public:
         Phases() noexcept = default;
-        Phases( bool oil, bool gas, bool wat, bool solvent = false, bool polymer = false, bool energy = false ) noexcept;
+        Phases( bool oil, bool gas, bool wat, bool solvent = false, bool polymer = false, bool energy = false,
```

Comment:
Something is heading a bit astray here, when the phases used in 99% of the cases - i.e. oil, gas and water, constitute a diminishing part of the code volume and complexity. But that that is probably not for this PR.

Reply:
We should distinguish oil, gas and water from other added components.  They are the central part of the black-oil based simulator.